functions.php

```diff
@@ -78,8 +78,13 @@ function stock_setup() {
  */
 function stock_get_default_colors() {
 	return array(
-		'primary'   => '#000000',
-		'secondary' => '#f5f5f5',
+		'primary'   			=> '#000000',
+		'secondary' 			=> '#f5f5f5',
+		'text' 					=> '#404040',
+		'input_background'		=> '#ffffff',
+		'input_text'			=> '#666666',
+		'input_focus'			=> '#222222',
+		'entry_title'			=> '#000000',
```

Comment:
Please change these `tab` characters to spaces, aligned with the longest key.

See the _Exception_ listed here: https://make.wordpress.org/core/handbook/coding-standards/php/#indentation